<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimingSettings</class>
 <widget class="QDialog" name="TimingSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-color: rgb(248, 255, 246);</string>
  </property>
  <widget class="QPushButton" name="pb_timingStop">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>定时停止</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_timingPlay">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>定时播放</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_timingClose">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>定时关机</string>
   </property>
  </widget>
  <widget class="TimingStop" name="wgt_timingStop" native="true">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>400</width>
     <height>310</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>511</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 255, 0);</string>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>491</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>定时设置</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pb_close">
       <property name="maximumSize">
        <size>
         <width>31</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="TimingPlay" name="wgt_timingPlay" native="true">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>400</width>
     <height>310</height>
    </rect>
   </property>
  </widget>
  <widget class="TimingClose" name="wgt_timingClose" native="true">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>400</width>
     <height>310</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TimingStop</class>
   <extends>QWidget</extends>
   <header>timingstop.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TimingPlay</class>
   <extends>QWidget</extends>
   <header>timingplay.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TimingClose</class>
   <extends>QWidget</extends>
   <header location="global">timingclose.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
